src/visualization/dgmulti_plot_utilities.jl

```diff
@@ -0,0 +1,107 @@
+"""
```

Comment:
I guess this file is largely copied from #613, isn't it? Since I have commented on that one, it might be easier to finish #613 first before polishing this PR

Reply:
Agreed. I'm thinking to first merge https://github.com/trixi-framework/Trixi.jl/pull/613, then merge those changes into this PR.